### PR TITLE
no more vars in Web/API (3)

### PR DESCRIPTION
**Pull request by lionralfs**
#### Summary

Replaces `var` in Web/API with `const` where applicable and `let` otherwise.
Mostly automated via my ESLint setup, using the rules `no-var` and `prefer-const`

#### Motivation

* See #16662
* See https://github.com/mdn/content/discussions/10389
* See https://github.com/orgs/mdn/discussions/143

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/element/outerhtml/index.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Also, while the element will be replaced in the document, the variable whose
 element:
 
 ```js
-var p = document.getElementsByTagName("p")[0];
+const p = document.getElementsByTagName("p")[0];
```
Comment (teoli2003) [inline files/en-us/web/api/element/outerhtml/index.md:113]:
```suggestion
const p = document.getQuerySelector("p");
```

File: files/en-us/web/api/filesystemhandle/issameentry/index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The **`isSameEntry()`** method of the
 ## Syntax
 
 ```js
-var Boolean = FileSystemHandle1.isSameEntry(FileSystemHandle2);
+const Boolean = FileSystemHandle1.isSameEntry(FileSystemHandle2);
```
Comment (teoli2003) [inline files/en-us/web/api/filesystemhandle/issameentry/index.md:22]:
```suggestion
isSameEntry(fileSystemHandle)
```

File: files/en-us/web/api/filesystemhandle/querypermission/index.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ current handle.
 ## Syntax
 
 ```js
-var PermissionState = FileSystemHandle.queryPermission(FileSystemHandlePermissionDescriptor);
+const PermissionState = FileSystemHandle.queryPermission(FileSystemHandlePermissionDescriptor);
```
Comment (teoli2003) [inline files/en-us/web/api/filesystemhandle/querypermission/index.md:22]:
```suggestion
queryPermission(FileSystemHandlePermissionDescriptor)
```

File: files/en-us/web/api/filesystemhandle/requestpermission/index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ file handle.
 ## Syntax
 
 ```js
-var PermissionState = FileSystemHandle.requestPermission(FileSystemHandlePermissionDescriptor);
+const PermissionState = FileSystemHandle.requestPermission(FileSystemHandlePermissionDescriptor);
```
Comment (teoli2003) [inline files/en-us/web/api/filesystemhandle/requestpermission/index.md:22]:
```suggestion
requestPermission(FileSystemHandlePermissionDescriptor)
```

File: files/en-us/web/api/htmliframeelement/contentdocument/index.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ If the iframe and the iframe's parent document are [Same Origin](/en-US/docs/Web
 ## Example of contentDocument
 
 ```js
-var iframeDocument = document.getElementsByTagName("iframe")[0].contentDocument;
+const iframeDocument = document.getElementsByTagName("iframe")[0].contentDocument;
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/contentdocument/index.md:15]:
```suggestion
const iframeDocument = document.getQuerySelector("iframe").contentDocument;
```

File: files/en-us/web/api/htmliframeelement/contentwindow/index.md [hunk 6]
```diff
@@ -23,7 +23,7 @@ A [Window](/en-US/docs/Web/API/Window) object.
 ## Examples
 
 ```js
-var x = document.getElementsByTagName("iframe")[0].contentWindow;
+const x = document.getElementsByTagName("iframe")[0].contentWindow;
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/contentwindow/index.md:26]:
```suggestion
const x = document.getQuerySelector("iframe").contentWindow;
```

File: files/en-us/web/api/htmliframeelement/contentwindow/index.md [hunk 7]
```diff
@@ -23,7 +23,7 @@ A [Window](/en-US/docs/Web/API/Window) object.
...
 //x = window.frames[0];
 
 x.document.getElementsByTagName("body")[0].style.backgroundColor = "blue";
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/contentwindow/index.md:29]:
```suggestion
x.document.getQuerySelector("body").style.backgroundColor = "blue";
```

File: files/en-us/web/api/htmliframeelement/referrerpolicy/index.md [hunk 8]
```diff
@@ -55,10 +55,10 @@ resource.
 ## Examples
 
 ```js
-var iframe = document.createElement("iframe");
+const iframe = document.createElement("iframe");
 iframe.src = "/";
 iframe.referrerPolicy = "unsafe-url";
-var body = document.getElementsByTagName("body")[0];
+const body = document.getElementsByTagName("body")[0];
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/referrerpolicy/index.md:61]:
```suggestion
const body = document.getQuerySelector("body");
```

File: files/en-us/web/api/htmliframeelement/src/index.md [hunk 9]
```diff
@@ -21,9 +21,9 @@ iframeElt.src= refStr;
 ## Example
 
 ```js
-var iframe = document.createElement("iframe");
+const iframe = document.createElement("iframe");
 iframe.src = "/";
-var body = document.getElementsByTagName("body")[0];
+const body = document.getElementsByTagName("body")[0];
```
Comment (teoli2003) [inline files/en-us/web/api/htmliframeelement/src/index.md:26]:
```suggestion
const body = document.getQuerySelector("body");
```

File: files/en-us/web/api/htmllinkelement/rel/index.md [hunk 10]
```diff
@@ -29,9 +29,9 @@ A string.
 ## Examples
 
 ```js
-var links = document.getElementsByTagName('link');
-var length = links.length;
-for (var i = 0; i < length; i++) {
+const links = document.getElementsByTagName('link');
+const length = links.length;
+for (let i = 0; i < length; i++) {
   alert(links[i]);
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/rel/index.md:35]:
```suggestion
for (const link of links) {
  console.log(link);
```

File: files/en-us/web/api/htmlinputelement/labels/index.md [hunk 11]
```diff
@@ -37,7 +37,7 @@ with the `<input>` element.
 ```js
 window.addEventListener("DOMContentLoaded", function() {
   const input = document.getElementById("test");
-  for(var i = 0; i < input.labels.length; i++) {
+  for(let i = 0; i < input.labels.length; i++) {
     console.log(input.labels[i].textContent); // "Label 1" and "Label 2"
```
Comment (teoli2003) [inline files/en-us/web/api/htmlinputelement/labels/index.md:41]:
```suggestion
  for (const label of input.labels) {
    console.log(label.textContent); // "Label 1" and "Label 2"
```

File: files/en-us/web/api/htmlmeterelement/labels/index.md [hunk 12]
```diff
@@ -36,7 +36,7 @@ with the `<meter>` element.
 ```js
 window.addEventListener("DOMContentLoaded", function() {
   const meter = document.getElementById("test");
-  for(var i = 0; i < meter.labels.length; i++) {
+  for(let i = 0; i < meter.labels.length; i++) {
     console.log(meter.labels[i].textContent); // "Label 1" and "Label 2"
```
Comment (teoli2003) [inline files/en-us/web/api/htmlmeterelement/labels/index.md:40]:
```suggestion
  for (const label of meter.labels) {
    console.log(label.textContent); // "Label 1" and "Label 2"
```

File: files/en-us/web/api/htmlprogresselement/labels/index.md [hunk 13]
```diff
@@ -36,7 +36,7 @@ with the `<progress>` element.
 ```js
 window.addEventListener("DOMContentLoaded", function() {
   const progress = document.getElementById("test");
-  for(var i = 0; i < progress.labels.length; i++) {
+  for(let i = 0; i < progress.labels.length; i++) {
     console.log(progress.labels[i].textContent); // "Label 1" and "Label 2"
```
Comment (teoli2003) [inline files/en-us/web/api/htmlprogresselement/labels/index.md:40]:
```suggestion
  for (const label of progress.labels) {
    console.log(label.textContent); // "Label 1" and "Label 2"
```

File: files/en-us/web/api/htmloutputelement/labels/index.md [hunk 14]
```diff
@@ -36,7 +36,7 @@ with the `<output>` element.
 ```js
 window.addEventListener("DOMContentLoaded", function() {
   const output = document.getElementById("test");
-  for(var i = 0; i < output.labels.length; i++) {
+  for(let i = 0; i < output.labels.length; i++) {
     console.log(output.labels[i].textContent); // "Label 1" and "Label 2"
```
Comment (teoli2003) [inline files/en-us/web/api/htmloutputelement/labels/index.md:40]:
```suggestion
  for (const label of output.labels) {
    console.log(label.textContent); // "Label 1" and "Label 2"
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few side changes

File: files/en-us/web/api/element/outerhtml/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/api/element/outerhtml/index.md:113]:
```suggestion
const p = document.querySelector("p");
```

?

Comment (teoli2003) [reply files/en-us/web/api/element/outerhtml/index.md:113]: Oh yes, of course.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**